cmd/cartesi-rollups-cli/root/db/check/check.go

```diff
@@ -17,13 +17,12 @@ var Cmd = &cobra.Command{
 }
 
 func run(cmd *cobra.Command, args []string) {
-
-	schemaManager, err := repository.NewSchemaManager(common.PostgresEndpoint)
+	schema, err := schema.New(common.PostgresEndpoint)
 	cobra.CheckErr(err)
-	defer schemaManager.Close()
+	defer schema.Close()
 
-	err = schemaManager.ValidateSchemaVersion()
+	version, err := schema.ValidateVersion()
 	cobra.CheckErr(err)
 
-	fmt.Printf("Database Schema is at the correct version: %d\n", repository.EXPECTED_VERSION)
+	slog.Info("Database Schema is at the correct version.", "version", version)
```

Comment:
There were requests to use fmt.Printf instead of slog as the CLI comands output. But we can decide and standardize it on another PR